examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
Ok I've made changes to use vertex colors and tested gles2 and 3 all working. Sorry about that, nice it's working closer desktop now though.